fairscale/nn/wrap/auto_wrap.py

```diff
@@ -34,26 +83,25 @@ def enable_wrap(module_blocklist: Optional[List] = None, **wrapper_kwargs: Any)
         with enable_wrap(**params):
             # Wraps layer in FSDP by default if within context
             self.l1 = wrap(torch.nn.Linear(5, 5))
-            # Wraps children modules by default based on min_num_params
-            self.l2 = auto_wrap(TransformerBlock(), min_num_params=1e8)
+            # Wraps children modules based on a different min_num_params
+            my_should_wrap = functools.partial(should_wrap, min_num_params=1e7)
+            self.l2 = auto_wrap(TransformerBlock(), shuold_wrap=my_should_wrap)
 
     Args:
-        module_blocklist: List of additional Module Classes to not wrap when
```

Comment:
module blocklist is no longer needed since we can do all customization with should_wrap now.